packages/eslint-plugin/src/index.ts

```diff
@@ -26,7 +26,7 @@ export const rules = {
   'no-keyframes-tagged-template-expression': noKeyframesTaggedTemplateExpressionRule,
   'no-styled-tagged-template-expression': noStyledTaggedTemplateExpressionRule,
   'no-suppress-xcss': noSuppressXCSS,
-  'no-empry-styled-expression': noEmptyStyledExpressionRule,
+  'no-empty-styled-expression': noEmptyStyledExpressionRule,
```

Comment:
This will need a bugfix changeset since it has already been released with the typo

Reply:
ah yeah, you'll need to run `yarn changeset` (with `patch` instead of `minor`) cuz we want to release a new version

Reply:
Amazing, thank you for the fast review :)